src/phoneField/specifications/default.md

```diff
@@ -0,0 +1,42 @@
+# Default PhoneField
+
+```js
+import { PhoneField } from '@blablacar/ui-library'
+```
+
+## Playground
+
+<!-- STORY -->
+
+## Specifications
+
+This field consists of the input select field and regular input field.
+
+> Phone input
+
+Phone input part should only accept digits and trigger numeric keyboards on mobile devices.
+
+> Country phone code
+
+The country code should be displayed next to a country flag when selection is made.
+On the country list the full name plus code should be visible.
+
+## Default usage
+
+```js
+<PhoneField name="phoneField" onChange={() => {}} />
+```
+
+## Behaviour
+
+On small devices on SPA the country selector and the phone input field should get separated into two separate fields.
```

Comment:
SPA might not mean much for external users. Not sure how much we care though.

Reply:
Yeah I'l adapt a bit this content.